symphony.rb

```diff
@@ -0,0 +1,74 @@
+class Symphony < Formula
+  desc "Framework for solving mixed integer linear programs"
+  homepage "https://projects.coin-or.org/SYMPHONY"
+  url "http://www.coin-or.org/download/pkgsource/SYMPHONY/SYMPHONY-5.6.14.tgz"
+  sha256 "666d1c72c4bb084b470c4f066b1694f7b3b11a479de15d9d1e9bfe1b53e3ae63"
+  head "https://projects.coin-or.org/svn/SYMPHONY/trunk"
+
+  option "without-openmp", "Disable openmp support"
```

Comment:
We have to use `needs :openmp if build.with? "openmp"` so Homebrew selects an appropriate compiler (clang doesn't support OpenMP yet).